src/aaudio/AAudioLoader.cpp

```diff
@@ -38,7 +38,7 @@ int AAudioLoader::open() {
     if (mLibHandle != nullptr) {
         return 0;
     }
-    mLibHandle = dlopen(LIB_AAUDIO_NAME, 0);
+    mLibHandle = dlopen(LIB_AAUDIO_NAME, RTLD_LAZY);
```

Comment:
Investigate whether this could have a real time impact

Reply:
Copying my reply from https://github.com/google/oboe/pull/352#discussion_r252294750:

> Sorry, I might've been mistaken, but I thought that `RTLD_LAZY` is `0`, so my point was to use explicit identified instead of a literal value.

Now I have checked and see that `RTLD_LOCAL` is for `0`, so I guess either of those is fine, depending on the performance.